package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npm i && cd ./client/ && npm i && cd ../ && cd ./server/ && npm i && cd ../ && npm run dev",
+    "start": "npm run dev",
+    "setup": "npm i && cd ./client/ && npm i",
```

Comment:
The earlier `start` command also installed server dependencies. And started both client and server.

If you want to add a `setup` script, remove the starting part and keep the process as it is. Plus, inside `start` script use `setup` script and `dev` script together.

Reply:
thats the thing one does not want to install dependencies everytime one changes anything.
Imagine you've just changed App.css it would trigger a npm install  that seems useless

Reply:
@prophet1996 I think you also need to add `cd ./server/ && npm i`. 
Currently `npm run setup` will only set up the client folder and not the server folder.

Reply:
So, yeah exactly. Let's keep the `setup` part but install both client and server dependencies when someone runs setup. Let `start` focus on only starting the application.